### PR TITLE
[Mobile-7370] Release notes for android agent v6.4.0

### PR DESCRIPTION
**Pull request by ywang-nr**
* What problems does this PR solve? - release notes for android agent v6.4.0

### CODE REVIEW

**Comment by ywang-nr** [comment]
@cthomas-newrelic for review

**Comment by rhetoric101** [comment]
@ywang-nr, it looks like this is a draft PR that is getting reviewed by your peers, right? Could you ping our hero in the documentation channel when you're ready for a writing review?

**Comment by ywang-nr** [comment]
@rhetoric101 yes, sure thing. I will comment here once it's ready for PROD. Thank you for checking.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+subject: Android agent
+releaseDate: '2021-12-07'
+version: 6.4.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_6.4.0.zip'
+---
+
+
+## Fixed in this release
+* Resolved D8/R8 warning messages: Expected stack map table for method with non-linear control flow.
+* Fixed a crash casued by NoSuchMethodError happened on Andorid OS 5 and OS 6
```
Comment (barbnewrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx:11]: "Andorid" s/be Android. Also recommend adding a period at end of sentence like previous bullet.

**Comment by barbnewrelic** [comment]
@ywang-nr  I noticed that your PR is waiting for a review from @cthomas-newrelic . No rush, I just wanted you to be aware that your PR also did not build correctly. We will keep your PR in draft state until you're ready to proceed. Or, let us know if you prefer for us to simply close this PR. Thanks for confirming! No rush.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_6.4.0.zip'
+---
+
+
+## Fixed in this release
+* Resolved D8/R8 warning messages: Expected stack map table for method with non-linear control flow.
+* Fixed a crash casued by NoSuchMethodError happened on Android OS 5 and OS 6.
+
+## Support statement:
+New Relic recommends that you upgrade the agent regularly and at a minimum every 3 months. As of this release, the oldest supported version is [5.23.0](https://docs.newrelic.com/docs/release-notes/mobile-release-notes/android-release-notes/android-5230).
```
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx:14]: Let's update the oldest supported to `5.24.0`

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+subject: Android agent
+releaseDate: '2022-01-12'
+version: 6.4.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_6.4.0.zip'
+---
+
+
+## Fixed in this release
+* Resolved D8/R8 warning messages: Expected stack map table for method with non-linear control flow.
+* Fixed a crash casued by NoSuchMethodError happened on Android OS 5 and OS 6.
```
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx:11]: Typo: `caused`.

Perhaps rephrase as "_Fixed crash caused by NoSuchMethodError that occurred on Android 5 and 6._"

**Review by cthomas-newrelic — CHANGES_REQUESTED** [review]

**Review by cthomas-newrelic — DISMISSED** [review]

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx (hunk 1 above)
Comment (paperclypse) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-640.mdx]: Looks like this got fixed

**Review by paperclypse — DISMISSED** [review]
Looks good

**Comment by paperclypse** [comment]
Hold for release until Jan 12, 2022

**Review by paperclypse — APPROVED** [review]
Looks good. Thanks @barbnewrelic!

**Comment by paperclypse** [comment]
Now that the afternoon merge is done, this will go out with tomorrow morning's merge to main.

**Pull request merged**